src/Visualization/Python/InterpolateVolumeData.py

```diff
@@ -0,0 +1,342 @@
+        type=int,
+        default=None,
+        help=("The maximum number of processes to be started. "
+              "A process is spawned for each source file up to this number."))
+
+    parser.add_argument('-v',
+                        '--verbose',
+                        action='count',
+                        default=0,
+                        help="Verbosity (-v, -vv, ...)")
+
+    return parser.parse_args(sys_args)
+
+
+if __name__ == "__main__":
+
+    parsed_args = parse_args(sys.argv[1:])
+
+    logging.basicConfig(level=logging.WARNING - parsed_args.verbose * 10)
+    source_files = glob.glob(parsed_args.source_file_prefix + "[0-9]*.h5")
+    file_numbers = [
+        re.match(parsed_args.source_file_prefix + "([0-9]*).h5",
+                 source_file).group(1) for source_file in source_files
+    ]
+    target_files = [
+        "{}{}.h5".format(parsed_args.target_file_prefix, number)
+        for number in file_numbers
+    ]
+
+    if len(source_files) == 0:
+        raise NameError("No files found matching the input pattern.")
+
+    # construct target mesh
+    target_basis = basis_from_string(parsed_args.target_basis) if len(
+        parsed_args.target_basis) > 1 else basis_from_string(
+            parsed_args.target_basis[0])
+    target_quadrature = quadrature_from_string(
+        parsed_args.target_quadrature) if len(
+            parsed_args.target_quadrature) > 1 else quadrature_from_string(
+                parsed_args.target_quadrature[0])
+    target_extents = parsed_args.target_extents[0] if len(
+        parsed_args.target_extents) == 1 else parsed_args.target_extents
+
+    # get dimension from first file
+    source_file = spectre_h5.H5File(source_files[0], "a")
+    source_vol = source_file.get_vol(parsed_args.source_subfile_name)
+    dim = source_vol.get_dimension()
+    target_mesh = Mesh[dim](target_extents, target_basis, target_quadrature)
+
+    interpolate_args = []
+    logging.info("Source and target files/volumes are as follows:")
+
+    for source_file, target_file in zip(source_files, target_files):
+        interpolate_args.append(
+            dict(source_file_path=source_file,
+                 target_mesh=target_mesh,
+                 target_file_path=target_file,
+                 source_volume_data=parsed_args.source_subfile_name,
+                 target_volume_data=parsed_args.target_subfile_name,
+                 components_to_interpolate=parsed_args.tensor_components,
+                 obs_start=parsed_args.start_time,
+                 obs_end=parsed_args.stop_time,
+                 obs_stride=parsed_args.stride))
+
+        logging.info("{}{} => {}{}".format(source_file,
+                                           parsed_args.source_subfile_name,
+                                           target_file,
+                                           parsed_args.target_subfile_name))
+
+    # need function because lambdas cant be pickled
+    def forward_args(args):
```

Comment:
I'm getting this error on Py 3.8.6:

```py
AttributeError: Can't get attribute 'forward_args' on <module 'spectre.Visualization.InterpolateVolumeData' from 'bin/python/spectre/Visualization/InterpolateVolumeData.py'>
```

I think the `forward_args` function must be top-level (and can be deleted once we can require Py3).

Reply:
doest that fix work for you? For me it works fine on 3.8.5, so I cannot imagine this is a version thing

Reply:
oh, youve got to be kidding me...

https://stackoverflow.com/questions/41385708/multiprocessing-example-giving-attributeerror

Reply:
how about we just use `starmap` and say the command line interface is for python3? python2 users can still use the function after all

Reply:
I think it's not as bad as the accepted answer in the stack overflow thread makes it sound. The function has to be importable, so just make it top-level (that fixed the issue for me). The bug/feature is only related to interactive interpreters (ipython) I think.

Reply:
ok, tell me if this works for you

Reply:
Works 👍 I can interpolate things to finite-difference grids now :) 
<img width="832" alt="Bildschirmfoto 2021-04-28 um 20 33 14" src="https://user-images.githubusercontent.com/746230/116455088-f70bbb00-a860-11eb-82ff-3480ba881252.png">